libraries/AP_BattMonitor/AP_BattMonitor_SMBus_Maxell.cpp

```diff
@@ -77,14 +77,14 @@ void AP_BattMonitor_SMBus_Maxell::timer()
 
     // read current (A)
     if (read_word(BATTMONITOR_SMBUS_MAXELL_CURRENT, data)) {
-        _state.current_amps = -(float)((int16_t)data) / 1000.0f;
+        _state.current_amps = -static_cast<float>(static_cast<int16_t>(data)) / 1000.0f * 2.0f;
```

Comment:
This should not have the static casts as well.

Reply:
I will change.